applications/GeoMechanicsApplication/tests/three_dimensional_piping/ProjectParameters.json

```diff
@@ -0,0 +1,192 @@
+{
+    "problem_data": {
+        "problem_name": "Cube_10sS_1",
+        "start_time": 0.0,
+        "end_time": 1.0,
+        "echo_level": 0,
+        "parallel_type": "OpenMP",
+        "number_of_threads": 1
+    },
+    "solver_settings": {
+        "solver_type": "Pw",
+        "model_part_name": "PorousDomain",
+        "domain_size": 3,
+        "model_import_settings": {
+            "input_type": "mdpa",
+            "input_filename": "Cube_10sS_1"
+        },
+        "material_import_settings": {
+            "materials_filename": "MaterialParameters.json"
+        },
+        "time_stepping": {
+            "time_step": 0.5,
+            "max_delta_time_factor": 1
+        },
+        "buffer_size": 2,
+        "echo_level": 0,
+        "clear_storage": false,
+        "compute_reactions": false,
+        "move_mesh_flag": false,
+        "reform_dofs_at_each_step": false,
+        "nodal_smoothing": false,
+        "block_builder": true,
+        "solution_type": "Steady_State_Groundwater_Flow",
+        "scheme_type": "Backward_Euler",
+        "reset_displacements": false,
+        "newmark_beta": 0.25,
+        "newmark_gamma": 0.5,
+        "newmark_theta": 0.5,
+        "rayleigh_m": 0.0,
+        "rayleigh_k": 0.0,
+        "strategy_type": "newton_raphson_with_piping",
+        "convergence_criterion": "water_pressure_criterion",
+        "displacement_relative_tolerance": 0.0001,
+        "displacement_absolute_tolerance": 1e-09,
+        "residual_relative_tolerance": 0.0001,
+        "residual_absolute_tolerance": 1e-09,
+        "water_pressure_relative_tolerance": 0.0001,
+        "water_pressure_absolute_tolerance": 1e-09,
+        "min_iterations": 6,
+        "max_iterations": 15,
+        "number_cycles": 100,
+        "reduction_factor": 0.5,
+        "increase_factor": 2.0,
+        "desired_iterations": 4,
+        "max_radius_factor": 10.0,
+        "min_radius_factor": 0.1,
+        "calculate_reactions": true,
+        "max_line_search_iterations": 5,
+        "first_alpha_value": 0.5,
+        "second_alpha_value": 1.0,
+        "min_alpha": 0.1,
+        "max_alpha": 2.0,
+        "line_search_tolerance": 0.5,
+        "rotation_dofs": true,
+        "linear_solver_settings": {
+            "solver_type": "LinearSolversApplication.skyline_lu_factorization",
+            "scaling": true
+        },
+        "problem_domain_sub_model_part_list": [
+            "Soil",
+            "Pipe"
+        ],
+        "processes_sub_model_part_list": [
+            "Min_X",
+            "Max_X",
+            "Soil"
+        ],
+        "body_domain_sub_model_part_list": [
+            "Soil",
+            "Pipe"
+        ],
+        "max_piping_iterations": 1000
```

Comment:
This is a lot. If know what the test needs to reach convergence, please place an only slightly higher number here. Otherwise a failing test may use endless resources here.

Reply:
Indeed. keep as is.